src/app/routes/cpsAsset/utils/getAdditionalPageData.js

```diff
@@ -0,0 +1,45 @@
+import { STORY_PAGE } from '#app/routes/utils/pageTypes';
+import { getAssetType } from './getAssetType';
+import fetchPageData from '../../utils/fetchPageData';
+import isEmpty from 'ramda/src/isEmpty';
+
+const pageTypeUrls = (assetType, service) => {
+  switch (assetType) {
+    case STORY_PAGE:
+      return [
+        {
+          name: 'mostRead',
+          path: `/${service}/mostread`,
+        },
+      ];
+    default:
+      return null;
+  }
+};
+
+const validateResponse = ({ status, json }, name) => {
+  if (status == 200 && !isEmpty(json)) {
```

Comment:
```suggestion
  if (status === 200 && !isEmpty(json)) {
```
Is the linting allowing a `==` here?

Reply:
Not sure how that slipped through linting :/